src/components/ui/Dialog.tsx

```diff
@@ -63,20 +63,30 @@ const DialogContent = forwardRef<
       {...props}
     >
       {children}
-      <DialogClose className={cn('absolute right-3 top-3')}>
+      {/* Move DialogClose to the header */}
+
+      {/* <DialogClose className={cn('')}>
         <Icon icon={CrossIcon} color="black" size={3.5} />
         <span className="sr-only">Close</span>
-      </DialogClose>
+      </DialogClose> */}
```

Comment:
Could you please remove the old code and any unnecessary comments related to it? Thank you.